CHANGELOG.unreleased.md

```diff
@@ -26,5 +26,6 @@
 >
 > Rule of thumb: add packages on top.
 
+- remove xo-server-cloud package
```

Comment:
Maybe in a dedicated section and with the reason, something like: `this package was useless for OpenSource installations because required a complete XOA environment`.

/cc @pdonias

Reply:
As long as we don't miss it.